doc/replication.md

```diff
@@ -195,3 +195,7 @@ self.replicator = [replicatorFactory oneWay:pushReplication error:&error];
 self.replicator.delegate = self.replicatorDelegate;
 [self.replicator startWithError:&error];
 ```
+
+## Customising HTTP requests made during replication
+
+This can be done with HTTP interceptors. These are documented [here](httpinterceptors.md).
```

Comment:
Doc links are absolute in the changelog and relative here - my guess is we should pick one style and stick with it